idlc_mir_passes/Cargo.toml

```diff
@@ -8,6 +8,6 @@ edition = "2021"
 [dependencies]
 idlc_ast = { path = "../idlc_ast" }
 idlc_ast_passes = { path="../idlc_ast_passes" }
-idlc_errors = { version = "0.1.0", path = "../idlc_errors" }
+idlc_errors = { version = "0.2.0", path = "../idlc_errors" }
```

Comment:
We can just remove the version information from all these and just rely on paths so we don't have to change it. All of them are versioned to the top level anyway